plugins/modules/aci_l3out_logical_node.py

```diff
@@ -49,6 +49,12 @@
     type: str
     choices: [ 'yes', 'no' ]
     default: 'yes'
+  loopback_address:
+    description:
+    - The loopback IP address.
+    - An empty string removes a configured loopback address.
```

Comment:
- A configured loopback address can be removed by passing an empty string (see Examples).

Reply:
I agree would be better